src/app/server/Server.cpp

```diff
@@ -353,12 +353,12 @@ CHIP_ERROR Server::Init(const ServerInitParams & initParams)
         }
     }
 
-#if CHIP_CONFIG_PERSIST_SUBSCRIPTIONS
-    ResumeSubscriptions();
-#endif
-
+    PlatformMgr().AddEventHandler(OnPlatformEventWrapper, 0);
```

Comment:
Might as well pass `reinterpret_cast<intptr_t>(this)` for the second arg, and get the server from the arg in the callback instead of relying on `Server::GetInstance()`....

Reply:
Right, done.